packages/relayer/src/rpc-relayer/index.ts

```diff
@@ -219,7 +219,8 @@ export class RpcRelayer implements Relayer {
         call: {
           walletAddress: signedTxs.intent.wallet,
           contract: signedTxs.entrypoint,
-          input: data
+          input: data,
+          chainId: 0, // TODO: how to you set it here?
```

Comment:
@klaidliadon should this be an optional field in RIDL then?

Or should we require users to pass chainId at all times?